config.php

```diff
@@ -124,7 +124,7 @@
                     }
                 }
             }
-            if (is_array($post['categories']) && in_array($category, $post['categories'])) {
+            if (isset($post['categories'])) {
```

Comment:
in_array is important to return only the defined category.
Without this will return all posts that have category.